docs/model_configuration.md

```diff
@@ -501,6 +501,59 @@ the device kind of the instance, for instance, KIND_CPU is "cpu", KIND_MODEL is
   ]
 ```
 
+### Rate Limiter Config
+
+Instance group optionally specifies [rate limiter](rate_limiter.md)
+config which controls how the rate limiter operates on the instances
+in the group. The rate limiter configuration includes the following
```

Comment:
What if rate limited is enabled and instance doesn't specify any rate limiter config... what is the default?